static/js/unread.js

```diff
@@ -531,8 +558,15 @@ export function update_message_for_mention(message) {
 
     if (is_unmuted_mention || message.mentioned_me_directly) {
         unread_mentions_counter.add(message.id);
+        add_message_to_unread_mention_topics(message.id);
     } else {
         unread_mentions_counter.delete(message.id);
+        remove_message_from_unread_mention_topics(message.id);
+    }
+
+    if (message.type === "stream") {
+        const topic_key = recent_topics_util.get_topic_key(message.stream_id, message.topic);
+        recent_topics_ui.inplace_rerender(topic_key);
```

Comment:
Does `inplace_rerender` know to be a noop if recent topics isn't visible?

Reply:
Yes